fastlane/Fastfile

```diff
@@ -1,26 +1,41 @@
 # frozen_string_literal: true
 
-default_platform(:ios)
+default_platform(:mac)
```

Comment:
Is this change intended?

Reply:
@AliSoftware Yep, I figured we'd want the default to be Mac instead of iOS. Is that incorrect?

Reply:
Oh right I didn't notice this was a repo only for macOS (as opposed to eg DayOne for which both platforms live in same repo) 😅

Initially I thought that maybe that change was made while you were debugging stuff and forgot to roll back or something, not understanding why we'd prefer one platform over the other, but now that I've realized this repo is _only_ for macOS that makes way more sense 😅